File: Week 4/lecture/phonebook.c
# include "..\..\cs50.h"
# include <stdio.h>
# include <string.h>

int main(void){
    //open csv file
    FILE *file = fopen("phonebook.csv", "a");
    if (!file)
    {
        return 1;
    }

    string name = getstring("Name: ");
    string number = getstring("Phone Number: ");

    // print to file
    fprintf(file, "%s, %s\n", name, number);

    //close file
    fclose(file);
}

File: Week 6/lecture/phonebook.py
from cs50 import get_string

people = dict()
people = {
    "Carter": "+1-617-495-1000",
    "David": "+1-949-468-2750",

}
# dictionary is hash table

name = get_string("Name: ")

if name in people:
    print(f"Number: {people[name]}")
